src/modules/navigation/NavigationTabView.js

```diff
@@ -1,62 +0,0 @@
-import React, {PropTypes} from 'react';
-import {
-  Animated,
-  NavigationExperimental as Navigation
-} from 'react-native';
-
-const NavigationTabView = React.createClass({
-  propTypes: {
-    router: PropTypes.func.isRequired,
-    navigationState: PropTypes.object.isRequired,
-    onNavigateBack: PropTypes.func.isRequired,
-    onNavigateCompleted: PropTypes.func.isRequired,
-    shouldRenderHeader: PropTypes.bool
-  },
-
-  getDefaultProps() {
-    return {shouldRenderHeader: true};
-  },
-
-  renderHeader(props) {
-    return (
-      <Navigation.Header
-        {...props}
-        onNavigateBack={this.props.onNavigateBack}
-        getTitle={state => state.key}
-      />
-    );
-  },
-
-  renderScene(props) {
-    return (
-      <Navigation.Card
-        {...props}
-        onNavigateBack={this.props.onNavigateBack}
-        key={props.scene.route.key}
-        renderScene={this.props.router}
-      />
-    );
-  },
-
-  render() {
-    return (
-      <Navigation.AnimatedView
```

Comment:
this will be deprecated in `0.30`. Also because of the refactory this view is no longer needed